devel/ci/integration/tests/fixtures/greenwave.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
I'll stop mentioning copyright and docblocks - let's just make sure all the things that need them have them.